ReactCommon/react/debug/react_native_assert.h

```diff
@@ -55,7 +55,7 @@ void react_native_assert_fail(
 #define react_native_assert(cond)                           \
   if (!(cond)) {                                            \
     LOG(ERROR) << "react_native_assert failure: " << #cond; \
-    google::FlushLogFiles(google::INFO);                    \
+    google::FlushLogFiles(google::GLOG_INFO);               \
```

Comment:
So this is where we use glog.. I was (separately) wondering if we actually needed to compile and link this 3rd party library for RN iOS/macOS.